Wrappers/Python/cil/optimisation/algorithms/Algorithm.py

```diff
@@ -119,10 +112,9 @@ def __next__(self):
         return self.iteration
 
     def _update_previous_solution(self):
-        """ Update the previous solution with the current one
+        r""" Update the previous solution with the current one
 
-        The concrete algorithm calls update_previous_solution. Normally this would
-        entail the swapping of pointers:
+        The concrete algorithm calls update_previous_solution. Normally this would entail the swapping of pointers:
```

Comment:
What does 'concrete algorithm' mean?

Reply:
I think it means a child class. I have rewritten this to make it clearer